src/main/java/lotto/domain/rank/LottoRevenueCalculator.java

```diff
@@ -0,0 +1,50 @@
+package lotto.domain.rank;
+
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LottoRevenueCalculator {
+
+    private Map<LottoRank, Integer> lottoRankCount = new LinkedHashMap<>(Map.of(
+            LottoRank.FIRST, 0,
+            LottoRank.SECOND, 0,
+            LottoRank.THIRD, 0,
+            LottoRank.FOURTH, 0,
+            LottoRank.FIFTH, 0
+    ));
+
+    private int totalSize;
+    private int totalMoney;
+
+    public LottoRevenueCalculator(List<LottoRank> lottoRanks) {
+        this.totalSize = addMoneyAndCount(lottoRanks);
+    }
```

Comment:
가능하면 생성자에서 인스턴스 변수를 초기화 하는게 어떨까요?
```suggestion
    public LottoRevenueCalculator(List<LottoRank> lottoRanks) {
        this.totalMoney = addMoneyAndCount(lottoRanks);
        this.totalSize = lottoRanks.size;
    }
```